File: Ulbra 2023-1/Introdução ao Desenvolvimento Web/Aula 17 (JavaScript)/exerc8.js
//8 - Faça um script que leia números até que seja digitado um número negativo, ao final mostrar a média dos números digitados. 

let numero = 0;
let soma = 0;
let contador = 0;

while (numero >= 0) {
  numero = Number(prompt("Digite um número (negativo para sair):"));
  if (numero >= 0) {
    contador++;
    soma += numero;
  }
}
if (contador > 0) {
  const media = soma / contador;
  alert("A média dos números digitados é: " + media);

} else {
  alert("Nenhum número foi digitado.");
}

File: Ulbra 2023-1/Algortimos e Programação/Aula 14 (Dinâmica For)/exerc8.py
x = 1
soma = 0
conta = 0 
primo = 0
number = 0

for i in range(12):
    num = float(input("Digite um numero: "))
    soma = soma + num
    media = soma / 12
    x += 1
    if conta == 0:
        menor = num
        maior = num
    if num > maior :
        maior = num
    elif menor > num :
        menor = num
    conta = conta + 1
    if num > 1:
        n = 0
    for i in range(12):
        if num % i == 0:
            print(num, 'não é primo')
            break
    else:
        soma += i
        print(num, 'é primo')

print(f"o menor numero é {menor}, o maior numero é {maior}, a média é {media} e a soma dos primos é {soma}")